server/api/urls.py

```
@@ -9,7 +9,8 @@
     path('users/<int:user_id>', UserIDView.as_view(), name='user-detail'),
     path('super',SuperView.as_view(), name='make edit super'),
     path('super/<int:super_id>', SuperIDView.as_view(),name='super-detail'),
-    
+    path('likes',LikeView.as_view(),name='likes'),
```

Ruff (0.8.2)

`LikeView` may be undefined, or defined from star imports

(F405)